qutip/entropy.py

```diff
@@ -244,24 +243,42 @@ def _entropy_relative(rho, sigma, base=e, sparse=False):
     rel_ent : float
         Value of relative entropy.
 
+    References
+    ----------
+
+    See Nielsen & Chuang, "Quantum Computation and Quantum Information",
+    Section 11.3.1, pg. 511 for a detailed explanation of quantum relative
+    entropy.
     """
     if rho.type != 'oper' or sigma.type != 'oper':
```

Comment:
Not your code, but I would prefer the property `isoper` to the test here.

Reply:
Thanks for the suggestion. Done!